web/components/application/hacker/pages/PageOne/PageOne.js

```diff
@@ -56,6 +83,108 @@ class PageOne extends Page {
           onBlur={() => this.setFieldAsBlurred('lastName')}
           error={this.getErrorIfBlurred('lastName')}
           />
+        <Select
+          options={genders}
+          value={gender}
+          name="gender"
+          label="What gender do you identify as?"
+          className="margin-ends-giga"
+          onChange={({ value: newGender }) => this.updateApplication({ gender: newGender })}
+          onBlur={() => this.setFieldAsBlurred('gender')}
+          error={this.getErrorIfBlurred('gender')}
+          isSearchable
+          allowNewOption
+          />
+        <RadioGroup
```

Comment:
Ay thanks for adding this, I think we're also missing Majors (it's in the application questions Notion page but not the mocks).

Reply:
For majors you can use[ what we used last year](https://github.com/nwplus/nwhacks2018_static/blob/8e38fe644e6dbc3e9bdfb248a86feb323ea9fdc9/data/specialties.txt), this looks pretty good too: https://github.com/fivethirtyeight/data/blob/master/college-majors/majors-list.csv